main.go

```diff
@@ -63,6 +63,7 @@ func main() {
 	mirrorURL := getopt.StringLong("mirror", 'm', defaultMirror, "Install from a remote API other than the default. Default: "+defaultMirror)
 	chDirPath := getopt.StringLong("chdir", 'c', dir, "Switch to a different working directory before executing the given command. Ex: tfswitch --chdir terraform_project will run tfswitch in the terraform_project directory")
 	versionFlag := getopt.BoolLong("version", 'v', "Displays the version of tfswitch")
+	defaultVersion := getopt.StringLong("default", 'd', defaultLatest, "Default to version to pick up. Ex: tfswitch --default 1.2.4")
```

Comment:
```suggestion
	defaultVersion := getopt.StringLong("default", 'd', defaultLatest, "Default to this version in case no other versions could be detected. Ex: tfswitch --default 1.2.4")
```

Reply:
@yermulnik In cases where TF_VERSION is already set (`say 1.2.4`) in the environment, if we add `TF_VERSION=1.2.3 tfswitch` it'll install version `1.2.3` which is not intended. But `tfswitch -d 1.2.3` can really act as a default version.